src/adapter/adapter.ts

```diff
@@ -80,6 +81,54 @@ abstract class Adapter extends events.EventEmitter {
             if (!serialPortOptions.path) {
                 throw new Error("No path provided and failed to auto detect path");
             }
+        } else if(serialPortOptions.path.indexOf("mdns://") != -1){
+            const mdnsDevice = serialPortOptions.path.substring(7);
+            if(mdnsDevice.length == 0){
+                throw new Error(
+                    `You must specify the adapter type after mdns://`+
```

Comment:
![z2m zeroconf format](https://user-images.githubusercontent.com/31830530/236251317-5e7b72ce-d8df-47f2-ba1a-bad6a7b3c076.png)
This is a misunderstanding, I meant MDNS service type. This error text really needs to be edited.

![image](https://user-images.githubusercontent.com/31830530/236252038-82a0590c-90ef-4657-b323-b5a930647241.png)
The user needs to specify only the **MDNS service type** of the device, all other configuration data will be received from the device via Zeroconf

Potentially, all Zeroconf devices listed [here ](https://github.com/home-assistant/core/blob/ef9bcd9285980385eede726e15aa1e7d20238eee/homeassistant/components/zha/manifest.json#L101)should work except **tube**
**tube** uses the **esphomelib** type, that is, if there are other esphome devices on the network, they will probably be found instead of the coordinator

Reply:
I see, this means that the config from the z2m `configuration.yaml` is ignored?

Reply:
True, some parameters from the config will be ignored because they will be received from the coordinator

Reply:
I see, this should be clearly stated in the docs, could you make a pr for the docs? (https://www.zigbee2mqtt.io/guide/configuration/adapter-settings.html)

Reply:
yes, I'm currently working on PR for docs